From be58dfbfaead0fbb65a97311f4836c06e3837b16 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 04 Jul 2018 16:02:17 +0300
Subject: [PATCH] board: habana: switch default to outlet #5

That outlet #5 is the most recent working configuration - make it
the default.
Note, this probably should somehow be defined in runtime/external configuration.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/ip_pdu b/board/habanalabs/common/ip_pdu
index ae351c6..a162367 100755
--- a/board/habanalabs/common/ip_pdu
+++ b/board/habanalabs/common/ip_pdu
@@ -13,7 +13,7 @@
 
 case "$board" in
 *asic*)
-	pwr_id=1
+	pwr_id=5
 	;;
 *onix)
 	pwr_id=1
